keras/preprocessing/image.py

```diff
@@ -838,6 +838,7 @@ class DirectoryIterator(Iterator):
             `"binary"`: binary targets (if there are only two classes),
             `"categorical"`: categorical targets,
             `"sparse"`: integer targets,
+            `"identical"`: image targets identical to input images,
```

Comment:
What would be some other keyword options? `identity`, `input`, `autoregression`?

Additionally, the explanation can be made clearer.